libs/web3/src/lib/use-get-withdraw-delay.ts

```diff
@@ -10,37 +10,66 @@ import { localLoggerFactory } from '@vegaprotocol/logger';
 
 const MAX_AGE = 5 * 60 * 1000; // 5 minutes
 
-type TimestampedDelay = { value: number | undefined; ts: number };
-const DELAY: TimestampedDelay = { value: undefined, ts: 0 };
+type TimestampedDelay = {
+  chainId: number;
+  value: number | undefined;
+  ts: number;
+};
+const DELAYS: TimestampedDelay[] = [];
```

Comment:
non blocking: If you're indexing by chainId, wouldn't using a map make it easier to manage the cache